certificate-policy.md

```diff
@@ -149,9 +167,9 @@ The FPKIPA shall affirm the suitability of any CPS to this policy.
 ### 1.5.4 CPS approval procedures
 A CPS shall be submitted and approved by the FPKIPA.
 
-Prior to submitting a CPS, the Issuing CA shall commission a compliance analysis study culminating in a written report that provides a summary of areas in which the CPS may not or does not comply with this CP. The compliance analysis shall be performed by an independent party. The CA shall resolve these discrepancies prior to submitting the CPS to the FPKIPA. The Issuing CA shall have an approved CPS, meet all CP and CPS requirements, conduct Federal Information Security Modernization Act assessment and authorization activities, and produce an authority to operate prior to commencing operations.
+Prior to submitting a CPS, the CA shall perform a compliance analysis culminating in a written report that provides a summary of areas in which the CPS may not or does not comply with this CP. The CA shall resolve these discrepancies prior to submitting the CPS to the FPKIPA. The CA shall have an approved CPS, meet all CP and CPS requirements, conduct Federal Information Security Modernization Act assessment and authorization activities, and produce an authority to operate prior to commencing operations.
```

Comment:
This removed "The compliance analysis shall be performed by an independent party." What was the rationale for that?

Reply:
Also, it's last minute so I won't insist on discussing this, but for the record, I don't think the CP should care at all about the ATO status of an issuing CA. That's a completely separate concern from whether certificates were properly issued. 

If a CA loses its ATO, whether it should be removed from the PKI hierarchy should be a decision of the FPKIPA, and not automatically a CP violation that we need to report to root stores.

Reply:
RE: removal of compliance analysis performed by independent party:
- request by teams to consider how many reviews already happen in our current processes 
- we have to define what an independent party is